src/pages/how-to/monitor-system-and-network-activity.mdx

```diff
@@ -1,44 +1,35 @@
 
-# Monitor system and network activity
 
-The activity monitoring feature lets you quickly see what's happening with your network.
-Whether a new machine or user joined your network or the access control policy has been modified, the activity log allows you to track the changes to your network.
+# Monitor System and Network Activity
 
-## Access activity monitoring view
+The activity monitoring functionality in NetBird allows you to observe and track changes to your network infrastructure. This includes events such as when a new machine or user has joined your network, or when an access control policy has been modified.
 
-Activity monitoring is enabled by default for every network, and you can access it in the web UI under the [Activity tab](https://app.netbird.io/activity).
-You can also use the search bar to filter events by activity type.
+## Access the Activity Monitoring View
+
+The activity monitoring feature is enabled by default for every NetBird network. You can access the activity log in the web UI under the [Activity tab](https://app.netbird.io/activity). This view provides a centralized log of network events, and you can use the search bar to filter events by activity type.
```

Comment:
Search bar is to search and filters to filter

Reply:
yeah good idea

Reply:
done